pom.xml

```diff
@@ -72,7 +72,7 @@
 								
 								<repository>
 									<id>m2eclipse</id>
-									<url>https://download.eclipse.org/technology/m2e/releases/</url>
+									<url>https://download.eclipse.org/technology/m2e/releases/1.19.0/</url>

```

Comment:
I think we want to rely always on the latest version or ?

Reply:
https://download.eclipse.org/technology/m2e/releases/latest/
And remove outdated version numbers

Reply:
No, we should always have a stable dependency to a fixed version. Otherwise our build piline might break if they release a new version and we don't even know which versions the piline used for the pipline runs....

Reply:
Under the link they keep only the newest releases so perhaps we should get it then from an archieve

Reply:
Well, you can still find 1.18.0 there which is not the newst. And if the remove the version from the link, then we know at least that our used version is not supported anymore and can manually update the M2E version